jupyterhub_traefik_proxy/etcd.py

```diff
@@ -63,8 +64,28 @@ def _default_client(self):
         help="""the etcd key prefix for traefik configuration""",
     )
 
+    traefik_api_url = Unicode(
+        "http://127.0.0.1:8099",
+        config=True,
+        help="""traefik authenticated api endpoint url""",
+    )
+
+    traefik_api_basic_auth_hashed_password = Unicode(
+        "$apr1$LD7Rp6hQ$QzRELO5C153vGLFIDorKJ0",
```

Comment:
This should probably not be configurable independently of the password itself. Instead, compute the hash from `self.traefik_api_basic_auth_password`